File: Programming-languages-and-laboratory/Python/Exercises/43_compito_java_20230221/Operatore.py
class Operatore:
    def __init__(self, codice, tipo, nome, costo_visita):
        self._codice = codice
        self._tipo = tipo
        self._nome = nome
        self._costo_visita = costo_visita

    def __str__(self):
        return self._nome + "\t" + str(self._codice) + "\t" + str(self._costo_visita)

File: Programming-languages-and-laboratory/Java/Exercises/43_compito_java_20230221/Operatore.java
public class Operatore {
    protected int codice;
    protected String tipo, nome_operatore;
    protected double costo_visita;

    public Operatore(int codice, String tipo, String nome_operatore, double costo_visita) {
        this.codice = codice;
        this.tipo = tipo;
        this.nome_operatore = nome_operatore;
        this.costo_visita = costo_visita;
    }

    public String toString() {
        return nome_operatore + "\t" + codice + "\t" + costo_visita;
    }
}